src/sentry/api/endpoints/project_details.py

```diff
@@ -202,6 +205,17 @@ def put(self, request, project):
             project.name = result['name']
             changed = True
 
+        if result.get('team'):
+            try:
+                project.team = Team.objects.filter(slug=result['team']).get()
```

Comment:
this would need socped to the current organization, and additionally need scoped to access the user has.

e.g.

```
request.access.has_team_scoope(team_we_found, 'project:write')
```

See also ``project_settings.py`` for how it enforces scope

Reply:
@dcramer It has been revised it in fbe56d1f3. How about now?